src/code/views/teacher-view.tsx

```diff
@@ -161,10 +175,18 @@ export class TeacherView extends React.Component<
                       simulation.stations.getStation(cell.weatherStationId);
       const precip = precipDiv(station);
       const city = showCities ? cityAnnotation(cell.weatherStationId) : null;
+      const cellLabel = cellName(cell.row, cell.column);
+      const group = groupMap[cellLabel];
+      const groupLabel = group
+                          ? <div className="grid-cell-group-label" style={styles.groupLabel}>
+                              {group || ''}
```

Comment:
At first glance this `||` check is redundant.

Reply:
Good catch; removed.